.pre-commit-config.yaml

```diff
@@ -78,27 +78,6 @@ repos:
             --linelength=88,
             '--filter=-readability/casting,-runtime/int,-build/include_subdir,-readability/fn_size'
         ]
--   repo: https://github.com/pycqa/pylint
```

Comment:
Reminder to remove :)

Reply:
yeah, done...

Could we move pylint to be run after mypy and pyright in pre-commit? It is really slow, so I try to avoid it...